server/storage/sql_models.go

```diff
@@ -11,6 +11,16 @@ const (
 	changeCategoryDeletion = "deletion"
 )
 
+// TUFFileTableName returns the name used for the tuf file table
+var TUFFileTableName = func() string {
```

Comment:
@ecordell I think these functions are the only thing that I really don't like in this PR. the rest of it I'm fine with (I've been waiting to see if you wanted to make the EndpointConfig updates here).

I really preferred them attached to the struct, it keeps it very clear what it's appropriate to use them for. However, if you want to break them out, it makes a lot more sense to just make them `const`'s.